.github/workflows/firestore.yml

```diff
@@ -30,13 +30,13 @@ jobs:
     # Only when this is not a scheduled run
     if: github.event_name != 'schedule'
     outputs:
-      changed: ${{ steps.changes.outputs.changed }}
+      changed: ${{ steps.changes.outputs.sources == 'true' && steps.changes.outputs.docs == 'false' }}
```

Comment:
Won't this also remove the requirement when CHANGELOG AND source files change?

Maybe better to iterate the directories to replace line 41?

Reply:
Nice catch! I tried one more thing that may be more future-proof than listing the directories.

Reply:
from https://github.com/dorny/paths-filter:

```yaml
- uses: dorny/paths-filter@v3
      id: filter
      with:
        # This makes it so that all the patterns have to match a file for it to be
        # considered changed. Because we have the exclusions for .jpeg and .md files
        # the end result is that if those files are changed they will be ignored
        # because they don't match the respective rules excluding them.
        #
        # This can be leveraged to ensure that you only build & test software changes
        # that have real impact on the behavior of the code, e.g. you can set up your
        # build to run when Typescript/Rust/etc. files are changed but markdown
        # changes in the diff will be ignored and you consume less resources to build.
        predicate-quantifier: 'every'
        filters: |
          backend:
            - 'pkg/a/b/c/**'
            - '!**/*.jpeg'
            - '!**/*.md'
```